config/management/src/lib.rs

```diff
@@ -284,14 +291,17 @@ pub mod tests {
             .unwrap()
             .public_key;
 
-        let validator_txn = validator_config(
+        validator_config(
             AccountAddress::random(),
             "/ip4/0.0.0.0/tcp/6180".parse().unwrap(),
```

Comment:
Can we refactor this into some sort of test constant string? It seems like it's used a lot of places, but iiuc it doesn't give much value other than just being a random address?